templates/discofuse/discofuse-sport/templates.yaml

```diff
@@ -0,0 +1,145 @@
+      \n {{incoherent_second_sentence}} \n{% endif %}"
+    name: decompose remainder 1
+    reference: ''
+    task_template: false
+  2f4a3f45-2367-495c-84ca-fee5833527b4: !Template
+    id: 2f4a3f45-2367-495c-84ca-fee5833527b4
+    jinja: 'Rewrite the following two sentences so that they flow better:
+
+
+      first sentence: {{incoherent_first_sentence}}
+
+
+      Second Sentence: {{incoherent_second_sentence}}
+
+
+      |||
+
+
+      {{coherent_first_sentence}} {{coherent_second_sentence}}'
+    name: fuse_instruction
+    reference: ''
+    task_template: false
+  3af62454-2938-4fff-ab0c-8083ba09b92b: !Template
+    id: 3af62454-2938-4fff-ab0c-8083ba09b92b
+    jinja: 'Here are two sentences:
+
+
+
+      1: {{incoherent_first_sentence}}
+
+
+      2: {{incoherent_second_sentence}}
+
+
+      Please edit them so that they sound more connected to each other, perhaps by
+      fusing the sentences together.
+
+      |||
+
+
+      {{coherent_first_sentence}} {{coherent_second_sentence}}'
+    name: fuse_instruction_2
+    reference: ''
+    task_template: false
+  6f1920ac-6b78-4892-8932-ccf92de5270d: !Template
+    id: 6f1920ac-6b78-4892-8932-ccf92de5270d
+    jinja: "{% if coherent_second_sentence==\"\" %}\nI'm doing some research, and\
+      \ found these facts:\n\n{{incoherent_first_sentence}} {{incoherent_second_sentence}}\
+      \ \n\nHow could I rewrite my facts to sound more natural?\n\n|||\n\n{{coherent_first_sentence}}\n\
+      {% endif %}"
+    name: fuse_4
+    reference: ''
+    task_template: false
+  73d198a5-9532-4894-9f26-3dccd60640ab: !Template
+    id: 73d198a5-9532-4894-9f26-3dccd60640ab
+    jinja: "{% if coherent_second_sentence==\"\" %}\n\nRewrite these two sentences\
+      \ as one sentence:\n\nFirst sentence: {{incoherent_first_sentence}} \n\nSecond\
+      \ sentence: {{incoherent_second_sentence}} \n\n|||\n\n{{coherent_first_sentence}}\n\
+      {% endif %}"
+    name: fuse_3
+    reference: ''
+    task_template: false
+  c9afa74a-c76f-4d8d-ac17-b3a477273a8e: !Template
+    id: c9afa74a-c76f-4d8d-ac17-b3a477273a8e
+    jinja: "{% if discourse_type != \"PAIR_NONE\" %}\n\nPassage A: {{coherent_first_sentence}}\
+      \ {{coherent_second_sentence}}\n\nPassage B: {{incoherent_first_sentence}} {{incoherent_second_sentence}}\n\
+      \nWhich of the following discourse phenomena have been removed in order to turn\
+      \ Passage A into Passage B?\n\n{{\"1: Apposition\"}}\n\n{{\"2: Relative Clauses\"\
+      }}\n\n{{\"3: Cataphora\"}}\n\n{{\"4: Verb Phrase Coordination\"}}\n\n{{\"5:\
+      \ Anaphora\"}}\n\n{{\"6: Inner Connectives\"}}\n\n{{\"7: Both Inner Connectives\
+      \ and Anaphora\"}}\n\n{{\"8: Sentence Coordination\"}}\n\n{{\"9 Both Sentence\
+      \ Coordination and Anaphora\"}}\n\n{{\"10: Forward Connectives\"}}\n\n{{\"11:\
+      \ Discourse Connectives\"}}\n\n{{\"12: Both Discourse Connectives and Anaphora\"\
+      }}\n\nAnswer with both the option letter and phenomena name.\n|||\n\n{{\n{\"\
```

Comment:
Same comment about predicting answer label vs. label + full answer string.

Reply:
sure! pushing a fix in a second